gradunwarp/core/tests/test_regression.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+from numpy.testing import assert_equal, assert_array_equal, \
+    assert_array_almost_equal, assert_almost_equal
+
+from gradunwarp.core import coeffs, utils
+from gradunwarp.core.unwarp_resample import siemens_B
+
+def test_siemens_B():
+    gradfile = 'gradunwarp/core/tests/data/gradunwarp_coeffs.grad'
+    siemens_coeffs = coeffs.get_coefficients('siemens', gradfile)
+    R0 = siemens_coeffs.R0_m  * 1000
+
+    vec = np.linspace(-300, 300, 60, dtype=np.float32)
+    x, y ,z = np.meshgrid(vec, vec, vec)
+
+    bx = siemens_B(siemens_coeffs.alpha_x, siemens_coeffs.beta_x, x, y, z, R0)
+    ref_bx = np.load('gradunwarp/core/tests/data/siemens_B_output.npz')['bx']
```

Comment:
This is going to be brittle, requiring the tests to be run with the working directory set to the repository root. `__file__` is not the cleanest way of referencing packaged files, but it works across all supported versions of Python:

```suggestion

import os

DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')

def test_siemens_B():
    gradfile = os.path.join(DATA_DIR, 'gradunwarp_coeffs.grad')
    siemens_coeffs = coeffs.get_coefficients('siemens', gradfile)
    R0 = siemens_coeffs.R0_m  * 1000

    vec = np.linspace(-300, 300, 60, dtype=np.float32)
    x, y ,z = np.meshgrid(vec, vec, vec)

    bx = siemens_B(siemens_coeffs.alpha_x, siemens_coeffs.beta_x, x, y, z, R0)
    ref_bx = np.load(os.path.join(DATA_DIR, 'siemens_B_output.npz'))['bx']
```